packages/esm-patient-appointments-app/translations/en.json

```diff
@@ -4,62 +4,30 @@
   "appointmentCancelError": "Error cancelling appointment",
```

Comment:
so these are (assumedly) all translations that no longer used in the patient-chart... however, assumedly most/all of them have been *added* to the patient management app... do we have a best practice/strategy, to copy above the translations from one to the other?

Reply:
Nothing that isn't "manually copy them" yet. Open to suggestions though!